### PR TITLE
add tiyan to the list of interns

### PR DESCRIPTION
**Pull request by tiyan-attirmidzi**
Preview URL is [Here](https://raw.githack.com/tiyan-attirmidzi/open-source/feature/add-tiyan/index.html)

### CODE REVIEW

File: en_ID/pages/profiles/tiyan.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Tiyan Attirmidzi
+
+Hi I'm [Tiyan Attirmidzi](https://github.com/tiyan-attirmidzi), you can call me Tiyan. I like software engineering because it is very cool! And my favorite food is fried noodles.
```
Comment (empeje) [inline en_ID/pages/profiles/tiyan.md]: Something more creative will be much more appreciated here 😄

**Review by empeje — CHANGES_REQUESTED** [review]

File: en_ID/pages/profiles/tiyan.md (hunk 1 above)
Comment (tiyan-attirmidzi) [reply en_ID/pages/profiles/tiyan.md]: @empeje well, I'll revise it :sweat_smile:

**Comment by tiyan-attirmidzi** [comment]
I have revised it @empeje :pray:

**Review by empeje — APPROVED** [review]
The PR looks good, just need to finish the series of first steps before moving forward.

**Comment by empeje** [comment]
Hi @tiyan-attirmidzi any difficulty on continuing the first steps?

**Comment by tiyan-attirmidzi** [comment]
Hi @empeje Previously, I apologize to you for being late in completing this first step. There is no real difficulty, only I am submitting the title of my final project. so I have to commute to campus, I will continue my first step and finish it.

**Comment by empeje** [comment]
Hi @tiyan-attirmidzi thank you for let us know. Please do let us know if there's anything that we can help with. Good luck with your final project!

**Comment by tiyan-attirmidzi** [comment]
thank you @empeje

**Comment by empeje** [comment]
Closed because of no activity.

**Pull request closed without merging**